Sources/BraveStrings/Resources/tr.lproj/BraveShared.strings

```diff
@@ -1888,6 +1900,9 @@
 /* Settings privacy section title */
 "Privacy" = "Gizlilik";
 
+/* The message displayed in alert when a user needs to set a passcode */
+"privacy.tab.tray.passcode.alert" = "Özel gezinme moduna geçmek için, önce cihazınıza bir parola belirlemeniz gerekmektedir.";
```

Comment:
"Özel gezinme moduna geçmek için, önce cihazınıza bir pin kodu belirlemeniz gerekmektedir.";